bodhi/tests/server/consumers/test_composer.py

```diff
@@ -1113,41 +1114,41 @@ def test_security_updates_parallel(self, publish, *args):
                      'ctype': 'rpm',
                      'updates': [u'bodhi-2.0-1.fc18'],
                      'agent': 'bowlofeggs'},
-                force=True, topic='mashtask.mashing'):
+                force=True, topic='composetask.composing'):
```

Comment:
I'll stop pointing these out, it's getting tedious ☺ sed can probably be used to find/fix them all.